test/index.js

```diff
@@ -48,6 +48,18 @@ test('Write dat.json to archive', function (t) {
   })
 })
 
+test('.create with no writable archive errors', function (t) {
+  var archive = { writable: false }
```

Comment:
Have to mock `archive` in these tests since it's not possible to force `writeable` to `false` on a real archive (`Object.defineProperty()` etc).